rotkehlchen/chain/evm/decoding/velodrome/velodrome_cache.py

```diff
@@ -197,7 +197,7 @@ def query_velodrome_data_from_chain_and_maybe_create_tokens(
 
     pool_data: list[dict] = []
     pool_data_chunk: list[dict] = []
-    offset, limit, last_notified_ts = 0, 200, Timestamp(0)
+    offset, limit, last_notified_ts = 0, 100, Timestamp(0)
```

Comment:
`reduced from 200 to 100 since some pools were getting skipped when hitting the limit. had to re-record the test.